t/Common.t

```diff
@@ -50,6 +50,16 @@ ok($sis->excludeLane(2), "Excluded lane 2");
 ok($sis->excludeLane(3), "Excluded lane 3");
 ok($sis->excludeLane(3), "Excluded lane 3");
 
+# Test if MD5 sum can be calculated
+my $md5 = $sis->getMd5("$testfolder1/sisyphus.yml", -noCache=>1);
+ok($md5 eq "ffe051d77d496c756e817f90ed5e905e", "MD5 sum was calculated");
+
+# Test if MD5 sum can be written to file
+$sis->saveMd5("$testfolder1/sisyphus.yml", $md5);
```

Comment:
No, true. But I does that NFSLock only work on NFS mounts? If so I think that it might be wise to refactor this code out into a class which can check if we have a NFS mount or not and lock the file in an appropriate manner depending on which case we are in.

Also - please note that I'm sort of nit-picking, so if @b97pla is happy with this we can just go ahead and merge.